File: chap08/run-model8-1.py
import pandas
import cmdstanpy

d = pandas.read_csv('input/data-salary-2.csv')
N = len(d)
data = {'N':N, 'X':d.X, 'Y':d.Y}

model = cmdstanpy.CmdStanModel(stan_file='model/model8-1.stan')
fit1 = model.sample(data=data, seed=123, parallel_chains=4)
fit1.save_csvfiles('output/result-model8-1')

File: chap08/run-model8-1.R
library(cmdstanr)

d <- read.csv('input/data-salary-2.csv')
N <- nrow(d)
data <- list(N=N, X=d$X, Y=d$Y)

model <- cmdstan_model(stan_file='model/model8-1.stan')
fit1 <- model$sample(data=data, seed=123, parallel_chains=4)
fit1$save_object(file='output/result-model8-1.RDS')
